supply-chain/config.toml

```diff
@@ -868,7 +868,7 @@ version = "0.2.0"
 criteria = "safe-to-deploy"
 
 [[exemptions.walkdir]]
-version = "2.3.2"
+version = "2.3.3"
```

Comment:
Could this stick to using 2.3.2 for now, or otherwise would you be ok auditing the 2.3.2 to 2.3.3 bump?